packages/atomizer/src/rules.js

```diff
@@ -4430,6 +4430,7 @@ module.exports = [
             {
                 bw: 'break-word',
                 n: 'normal',
+                a: 'anywhere',
```

Comment:
> To prevent overflow, an otherwise unbreakable string of characters — like a long word or URL — may be broken at any point if there are no otherwise-acceptable break points in the line. No hyphenation character is inserted at the break point. Soft wrap opportunities introduced by the word break are considered when calculating min-content intrinsic sizes.